spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/actions/RewritePositionDeleteFilesSparkAction.java

```diff
@@ -133,55 +133,59 @@ public RewritePositionDeleteFiles.Result execute() {
     }
   }
 
-  private Map<StructLike, List<List<PositionDeletesScanTask>>> planFileGroups() {
```

Comment:
Optional: A note to the above snippet. What about getting rid of `commitManager` var and just calling the method directly?

```
if (partialProgressEnabled) {
  return doExecuteWithPartialProgress(ctx, groupStream, commitManager());
} else {
  return doExecute(ctx, groupStream, commitManager());
}
```